src/fal/cli/dbt_runner.py

```diff
@@ -166,3 +160,117 @@ def _create_fal_result_file(target_path: str, run_index: int):
         shutil.copy(
             fal_run_result, os.path.join(target_path, f"fal_results_{run_index}.json")
         )
+
+
+def _process_logs(raw_logs: str):
```

Comment:
are the logs exactly the same as it would be if we invoked the `dbt` cli command